src/modules/compliance/src/lib/GenProcedureMap.pl

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env perl
```

Comment:
Can we not just use bash for this? Bringing in Perl just for this seems like overkill

Reply:
1. Perl is currently in all of our build environments. (unfortunately, Python is not).
2. We haven't decided finally yet, but we'll probably pre-generate the ProcedureMap and put it in the repo, so this script won't be launched at build time - only manually when someone adds/removes a procedure. For now, with the amount of churn, it's easier to just have it automated.

Reply:
Ohh nice, i didn't realize the environments already contained Perl, i just validated that ubun14.04/debian-10 have it 👌